.github/workflows/tests.yaml

```diff
@@ -34,21 +34,38 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
       - name: Load secrets
-        working-directory: tests/functional_tests
+        working-directory: tests/functional_tests/tools/virtualenv/keys
         run: |
           mkdir -p ~/.atsign/keys/
-          echo "${{ secrets.ATKEYS_12ALPACA }}" > ~/.atsign/keys/@12alpaca_key.atKeys
-          echo "${{ secrets.ATKEYS_12SNOWBOATING }}" > ~/.atsign/keys/@12snowboating_key.atKeys
+          ls -la
+          cp "@alice🛠_key.atKeys" ~/.atsign/keys/
+          cp "@bob🛠_key.atKeys" ~/.atsign/keys/
+
+      - name: Start Virtual Environment
+        working-directory: tests/functional_tests/tools/virtualenv
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y docker-compose
+          sudo ./start_virtualenv.sh
+          sleep 5
```

Comment:
Is it possible to test for the services being ready rather than sleeping?

Reply:
Sure, I'll open a ticket for this

Reply:
@JeremyTubongbanua see here: https://github.com/atsign-foundation/noports/tree/xc-docker-demo-env/tools/demo-env

Reply:
Should I implement docker readiness checking in this PR or should we merge this into trunk so it can be used by others now?

Reply:
I'm happy for it to be done later. So merge now.